_pytest/compat.py

```diff
@@ -21,6 +22,12 @@
     enum = None
 
 
+try:
+    from inspect import signature, Parameter as Parameter
```

Comment:
Please prefer to use explicit version checking instead of relying on `ImportError`: 

```python
if _PY3:
    from inspect import signature, Parameter as Parameter
else:
    from funcsigs import signature as signature, Parameter as Parameter
```

I've seen before a broken installation throwing an `ImportError`, which would go to the fallback statement and also fail, leaving me baffled for a awhile until finally figuring it out. 😅